ccloudvm/README.md

```diff
@@ -0,0 +1,26 @@
+# Test Kata using ccloudvm
+
+* [How to use Kata workloads for `ccloudvm`](#how-to-use-kata-workloads-for-ccloudvm)
+    * [Create Docker\* and Kata Containers virtualized enviroment](#create-docker-and-kata-containers-virtualized-enviroment)
+---
+
+The [ccloudvm](https://github.com/intel/ccloudvm/) tool is a command
+to create development and demo environments. The tool sets up these development
+environments inside a virtual machine.
+
+## How to use Kata workloads for `ccloudvm`
+
+- Follow the `ccloudvm` [install instructions](https://github.com/intel/ccloudvm/#introduction)
+
+### Create Docker\* and Kata Containers virtualized enviroment
+
+Create a virtual machine with Docker and Kata containers.
```

Comment:
s/containers/Containers/

Reply:
I think this should this be "Create a virtual machine with Docker and Kata Containers installed." ?